test/test-basics.js

```diff
@@ -56,6 +58,23 @@ test('json based media type', async () => {
   same(json.headers.accept, 'application/vnd.something.com')
 })
 
+test('manually-set content-type header when body is present', async () => {
```

Comment:
Let’s move this test and the import statement to an `else` at the bottom so that we only run it on Node.js. By default, all these tests will run in both Node.js and the browser.

Reply:
Ah, sorry, I didn’t see you had platform-specific tests at the end (and I didn’t want to introduce that if you didn’t). Done.